module_13/tests/login_test.py

```diff
@@ -0,0 +1,14 @@
+from selenium import webdriver
+from module_13.pages.login_page import LoginPage
+
+
+def test_login():
+    chrome_driver_path = "../driver/chromedriver-win64/chromedriver.exe"
```

Comment:
regarding your question in private messages, you can use your setup and teardown functions in few different ways in your frameworks, for example in pytest you can use fixtures
https://docs.pytest.org/en/6.2.x/fixture.html

PS: you will get all info about it in Pytest lecture btw

Reply:
Thank you! We just passed fixtures in the text module, with fixtures it's not a problem anymore :)